assets/js/amp-block-validation.js

```diff
@@ -42,7 +42,7 @@ var ampBlockValidation = ( function() { // eslint-disable-line no-unused-vars
 			wp.i18n.setLocaleData( module.data.i18n, 'amp' );
 
 			wp.hooks.addFilter(
-				'blocks.BlockEdit',
+				'editor.BlockEdit',
```

Comment:
This one is also captured in PR #1293 in [this commit](https://github.com/Automattic/amp-wp/pull/1293/commits/3b4e38184bf5f4113cd299d5f05ea26f6ca00756).  Just noting for reference.